installers/WindowsInstaller.hs

```diff
@@ -136,6 +139,8 @@ writeInstallerNSIS fullVersion = do
           , "DetailPrint \"liteFirewall::AddRule: $0\""
           ]
 
+        exec "build-certificates-win64.bat >build-certificates.log 2>&1"
```

Comment:
`exec` here is a NSIS directive to emit an instruction to execute the given command during install time -- so it seems to be right to me..